libs/mailroom/ngx/src/lib/modules/list/pages/detail/detail.component.scss

```diff
@@ -0,0 +1,59 @@
+.attachment {
```

Comment:
Same thing about `px` values. They are absolute values will not scale with the parent or root.

Reply:
Fixed.